docs/guides/recipe.rst

```diff
@@ -98,46 +134,183 @@ with the following snippet:
+
 
 This will add a reference to the ``honeybee-radiance`` Plugin, version
-``0.4.1`` which is cloud-hosted on Pollination's registry server. This will
+``0.5.0`` which is hosted on Pollination's registry server. This will
 allow us to use Functions which are defined in this Plugin as the
 implementation that does the work inside Tasks which are stitched together in
 the Recipe.
 
 Flow
 ----
 
-The ``flow/`` directory contains the YAML definitions of the DAGs of the recipe
-as YAML files. At a minimum, the directory must contain a ``main.yaml`` which
-is, surprisingly, the main graph.
+The ``flow`` directory is analogous to the ``src`` directory of a ``python``
+package. It is where the actual code of the Recipe is kept.
+
+Because the recipe files are verbose, we will simply link to them and examine
+some snippets. These should be saved with the name of the link inside the
+``flow`` directory.
+
+`main.yaml <https://storage.googleapis.com/lbt-blobs/documentation-samples/main.yaml>`_
+
+`annual-daylight-ray-tracing.yaml <https://storage.googleapis.com/lbt-blobs/documentation-samples/annual-daylight-ray-tracing.yaml>`_
+
+After downloading these, the directory should now look like this::
+
+    my-recipe
+    ├── flow
+    │   ├── main.yaml
+    │   └── annual-daylight-ray-tracing.yaml
+    ├── dependencies.yaml
+    ├── LICENSE
+    ├── package.yaml
+    └── README.md
 
 DAG Tasks
 ---------
 
-As mentioned above, a Task can itself be a DAG. This both allows complex graphs
-to be reused inside other graphs and allows isolating related Tasks into a
-logical unit.
-
-Add a new file at the path ``flow/dag.yaml``. Inside that file, place the
-following snippet:
+The first snippet that we will examine is the ``tasks`` key from ``main.yaml``.
+In the linked file that was saved in the previous section, it should begin like
+this:
 
 .. code-block:: yaml
 
-  type: DAG
-
-This file ties three key concepts together. The files itself defines that it is
-of ``type: DAG`` and has a ``tasks`` array of Task objects. The single Task
-that we define uses the function ``daylight-coefficient`` from the
-``honeybee-radiance`` Plugin that is defined in ``dependencies.yaml``.
+  tasks:
+  - type: DAGTask
+    annotations: {}
+    name: annual-daylight-raytracing
+    template: annual-daylight-ray-tracing
+    needs:
+    - create-sky-dome
+    - create-octree-with-suns
+    - create-octree
+    - generate-sunpath
+    - create-total-sky
+    - create-direct-sky
+    - create-rad-folder
+    arguments:
+    - type: TaskArgument
+      annotations: {}
+      name: sensor-count
+      from:
+        type: InputReference
+        annotations: {}
+        variable: sensor-count
+    - type: TaskArgument
+      annotations: {}
+      name: radiance-parameters
+      from:
+        type: InputReference
+        annotations: {}
+        variable: radiance-parameters
+    - type: TaskPathArgument
+      annotations: {}
```

Comment:
For the sake of readability we may want to remove some of these unset fields like annotations.